src/Ocelot.Provider.Polly/OcelotBuilderExtensions.cs

```diff
@@ -1,23 +1,34 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
+
 using Ocelot.Configuration;
 using Ocelot.DependencyInjection;
 using Ocelot.Errors;
 using Ocelot.Errors.QoS;
 using Ocelot.Logging;
 using Ocelot.Provider.Polly.Interfaces;
 using Ocelot.Requester;
+
 using Polly.CircuitBreaker;
 using Polly.Timeout;
 
 namespace Ocelot.Provider.Polly;
 
 public static class OcelotBuilderExtensions
 {
+    private static readonly Dictionary<Type, Func<Exception, Error>> ErrorMapping = new Dictionary<Type, Func<Exception, Error>>
+    {
+        {typeof(TaskCanceledException), e => new RequestTimedOutError(e)},
+        {typeof(TimeoutRejectedException), e => new RequestTimedOutError(e)},
+        {typeof(BrokenCircuitException), e => new RequestTimedOutError(e)},
+        {typeof(BrokenCircuitException<HttpResponseMessage>), e => new RequestTimedOutError(e)}
+    };
+
+
```

Comment:
I'm not a fan of anonymous delegates.
```suggestion
    public static readonly Dictionary<Type, Func<Exception, Error>> DefaultErrorMapping = new Dictionary<Type, Func<Exception, Error>>
    {
        {typeof(TaskCanceledException), CreateRequestTimedOutError},
        {typeof(TimeoutRejectedException), CreateRequestTimedOutError},
        {typeof(BrokenCircuitException), CreateRequestTimedOutError},
        {typeof(BrokenCircuitException<HttpResponseMessage>), CreateRequestTimedOutError},
    };

    private static Error CreateRequestTimedOutError(Exception e) => new RequestTimedOutError(e);

```